website/docs/deploy/configuring-unleash.md

```diff
@@ -203,7 +203,82 @@ function getLogger(name) {
+| `password`               | `DATABASE_PASSWORD`             | `passord`      | The database password.                                                                                                                                                                   |
+| `host`                   | `DATABASE_HOST`                 | `localhost`    | The database hostname.                                                                                                                                                                   |
+| `port`                   | `DATABASE_PORT`                 | `5432`         | The database port.                                                                                                                                                                       |
+| `database`               | `DATABASE_NAME`                 | `unleash`      | The name of the database.                                                                                                                                                                |
+| `ssl`                    | `DATABASE_SSL`                  | N/A            | An object describing SSL options. In code, provide a regular JavaScript object. When using the environment variable, provide a **stringified JSON object**.                              |
+| `pool`                   | N/A (use the below variables)  |                | An object describing database pool options. With environment variables, use the options below directly.                                                                                  |
+| `pool.min`               | `DATABASE_POOL_MIN`             | 0              | The minimum number of connections in the connection pool.                                                                                                                                |
+| `pool.max`               | `DATABASE_POOL_MAX`             | 4              | The maximum number of connections in the connection pool.                                                                                                                                |
+| `pool.idleTimeoutMillis` | `DATABASE_POOL_IDLE_TIMEOUT_MS` | 30000          | The amount of time (in milliseconds) that a connection must be idle for before it is marked as a candidate for eviction.                                                                 |
+|                          | `DATABASE_URL`                  | N/A            | A string that matches the [libpq connection string](https://www.postgresql.org/docs/current/libpq-connect.html#LIBPQ-CONNSTRING), such as `postgres://USER:PASSWORD@HOST:PORT/DATABASE`. |
+|                          | `DATABASE_URL_FILE`             | N/A            | The path to a file that contains a [libpq connection string](https://www.postgresql.org/docs/current/libpq-connect.html#LIBPQ-CONNSTRING).                                               |
```

Comment:
Are these supported in JS or are they env var only?

Reply:
They're in the JS as well, DATABASE_URL maps to databaseUrlFile and DATABASE_URL_FILE maps to databaseUrlFile, both in the top level options so:

const unleashOptions = {
  databaseUrlFile: "something",
  databaseUrl: "something-else"
  db: {
    user: 'unleash_user',
    password: 'passord',
   } 
}
 
Both env vars are ignored if their JS equivalent option is set in the config

Reply:
Thanks! I'm assuming you meant that `DATABASE_URL` maps to `databaseUrl` (as per your example) and not `databaseUrlFile`. 

Also a great point that in-code config overrides env vars: I'll make sure to specify that!

Reply:
@sighphyre Just to confirm: I've assumed that in-code config _always_ overrides env vars if they're both present. Is that correct or is that just for db url and url file?

Reply:
@thomasheartman It should be, yeah! There's no global force override for this so it's done on a case by case basis but I can't find a case where this isn't true